dashboard/dashboard-front/src/components/HeaderBar.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
+import { computed, type Ref } from 'vue'
+import { mdiAccountCircle, mdiLogout } from '@quasar/extras/mdi-v7'
+import { isAuthenticated } from '@/services/UserDataStore'
+import { envInfo } from '@/services/EnvironmentInfo'
+
+const emit = defineEmits(['toggleDrawer'])
+
+function toggleDrawer () {
+    emit('toggleDrawer', null)
+}
+
+const siteNamePrefix: Ref<string> = computed(() => {
+    if (!!envInfo.site_name) {
+        return `[${envInfo.site_name}]`
+    } else {
+        return ''
+    }
+})
+
+function getToolbarColor(siteName: string): string {
+    if (siteName == 'test') {
+        return '#1B5E20'
+    } else if (siteName == 'preprod') {
+        return '#EE6B00'
+    } else if (siteName == 'prod') {
+        return '#B71C1C'
+    } else {
+        return '#000000'
+    }
+}
+
+const toolbarStyle: Ref<string> = computed(() => {
+    const siteName = (envInfo.site_name || '').toLowerCase()
+    const toolbarColor = getToolbarColor(siteName)
+    return 'background-color: ' + toolbarColor + ';'
```

Comment:
Personally, I prefer `background-color: ${toolbarColor};` in cases like that but it's just a small thing.